app/assets/scripts/components/slate/plugins/equation/equation-editor.js

```diff
@@ -104,43 +109,50 @@ export default function EquationEditor(props) {
     }
   }, [latexEquation]);
 
-  // Focus equation input when user clicks the preview.
-  const onEquationPreviewClick = useCallback(
-    (e) => {
-      // When the equation preview box gets clicked we want to focus the
-      // equation field except if the user is selecting the equation characters.
-      const node = equationBlockRef.current;
-      if (node) {
-        const display = node.querySelector('.katex-display');
-        if (display.contains(e.target)) return;
-      }
-
-      // When trying to select something in the editor with a selection
-      // elsewhere (for example the equation characters) the slate editor
-      // crashed. To fix this before doing anything we use the Selection api to
-      // clear the existing selection (which is not managed by slate) and then
-      // let slate handle this from a clean slate. Pun intended.
-      window.getSelection().removeAllRanges();
-      const path = ReactEditor.findPath(editor, element);
-      Transforms.select(editor, Editor.end(editor, path));
-    },
-    [editor, element]
-  );
+  useLayoutEffect(() => {
+    textareaRef.current.style.height = '1px';
+    const scrollHeight = textareaRef.current.scrollHeight;
+    textareaRef.current.style.height = `${scrollHeight}px`;
+  }, [latexEquation]);
 
-  return readOnly ? (
-    <EquationPreviewBody {...attributes}>{equation}</EquationPreviewBody>
-  ) : (
-    <DeletableBlock deleteAction='delete-equation' {...attributes}>
-      <EquationInput spellCheck={false}>
-        <code>{children}</code>
-      </EquationInput>
+  const handleInputChange = (e) => setLatexEquation(e.target.value);
+  const handleSave = () => {
+    const path = element && ReactEditor.findPath(editor, element);
+    upsertEquation(editor, latexEquation, path);
+    editor.equationModal.reset();
+  };
+
+  const showInfo = () => {
+    editor.simpleModal.show({ id: 'latex-modal' });
+  };
+
+  return (
+    <>
+      <FormLabel id='equation'>
+        Enter Latex code
+        <Button
+          type='button'
+          useIcon='circle-information'
+          hideText
+          onClick={showInfo}
+        >
+          Latex cheatsheet
+        </Button>
+      </FormLabel>
+      <FormTextarea
+        ref={textareaRef}
+        id='equation'
+        name='equation'
+        value={latexEquation}
+        onChange={handleInputChange}
+        spellCheck={false}
+      />
```

Comment:
To match other fields you should use the [FormGroup structure](https://github.com/NASA-IMPACT/nasa-apt-frontend/blob/20b92a29b318ea4d48626e1b43ed62891d0b71e4/app/assets/scripts/components/common/forms/form-group-structure.js), or event the existing [textarea](https://github.com/NASA-IMPACT/nasa-apt-frontend/blob/9aede142f6c89b514902108243f68d61dbcc2b57/app/assets/scripts/components/common/forms/input-textarea.js) - it may need a couple of tweaks for min height